python/telemetry/aepex_sw_stat.py

```diff
@@ -0,0 +1,446 @@
+                                 NVM_SMALL15=15, NVM_SMALL16=16,
+                                 NVM_SMALL17=17, NVM_SMALL18=18,
+                                 NVM_SMALL19=19, NVM_SMALL20=20,
+                                 NVM_SMALL21=21, NVM_SMALL22=22,
+                                 NVM_SMALL23=23, NVM_SMALL24=24,
+                                 NVM_SMALL25=25, NVM_SMALL26=26,
+                                 NVM_SMALL27=27, NVM_SMALL28=28,
+                                 NVM_SMALL29=29, NVM_SMALL30=30,
+                                 NVM_SMALL31=31, NVM_SMALL32=32,
+                                 NVM_SMALL33=33, NVM_SMALL34=34,
+                                 NVM_SMALL35=35, NVM_SMALL36=36,
+                                 NVM_SMALL37=37, NVM_SMALL38=38,
+                                 NVM_SMALL39=39, NVM_SMALL40=40,
+                                 NVM_SMALL41=41, NVM_SMALL42=42,
+                                 NVM_SMALL43=43, NVM_SMALL44=44,
+                                 NVM_SMALL45=45, NVM_SMALL46=46,
+                                 NVM_SMALL47=47, NVM_SMALL48=48,
+                                 NVM_SMALL49=49, NVM_SMALL50=50,
+                                 NVM_SMALL51=51, NVM_SMALL52=52,
+                                 NVM_SMALL53=53, NVM_SMALL54=54,
+                                 NVM_SMALL55=55, NVM_SMALL56=56,
+                                 NVM_SMALL57=57, NVM_SMALL58=58,
+                                 NVM_SMALL59=59, NVM_SMALL60=60,
+                                 NVM_SMALL61=61, NVM_SMALL62=62,
+                                 NVM_SMALL63=63, NVM_SMALL64=64,
+                                 NVM_SMALL65=65, NVM_SMALL66=66,
+                                 NVM_SMALL67=67, NVM_SMALL68=68,
+                                 NVM_SMALL69=69, NVM_SMALL70=70,
+                                 NVM_SMALL71=71, NVM_SMALL72=72,
+                                 NVM_SMALL73=73, NVM_SMALL74=74,
+                                 NVM_SMALL75=75, NVM_SMALL76=76,
+                                 NVM_SMALL77=77, NVM_SMALL78=78,
+                                 NVM_SMALL79=79, NVM_LARGE0=80,
+                                 NVM_LARGE1=81, HOLDING0=82
+                                 ),
+)
+
+sw_partition = BitStruct(
+    'sw_store_partition_write_misc' / BitsInteger(32),
+    'sw_store_partition_read_misc' / BitsInteger(32),
+    'sw_store_partition_write_adcs' / BitsInteger(32),
+    'sw_store_partition_read_adcs' / BitsInteger(32),
+    'sw_store_partition_write_beacon' / BitsInteger(32),
+    'sw_store_partition_read_beacon' / BitsInteger(32),
+    'sw_store_partition_write_log' / BitsInteger(32),
+    'sw_store_partition_read_log' / BitsInteger(32),
+    'sw_store_partition_write_sci' / BitsInteger(32),
+    'sw_store_partition_read_sci' / BitsInteger(32),
+)
+
+sw_fp = BitStruct(
+    'sw_fp_resp_count' / BitsInteger(16),
+)
+
+sw_ana = BitStruct(
```

Comment:
`LinearAdapter` or `AffineAdapter` is suitable for many of these. A variable can be defined for the adapters of several fields that use the same coefficients (example: `PolynomialAdapter([0.000000e+00, 8.056600e-04], BitsInteger(16))`).